geocoding/example/lib/plugin_example/geocode_page.dart

```diff
@@ -75,7 +75,7 @@ class _GeocodeWidgetState extends State<GeocodeWidget> {
 
                     placemarkFromCoordinates(latitude, longitude)
                         .then((placemarks) {
-                      final output = placemarks[0].toString();
+                      final output = placemarks![0].toString();
 
                       print(placemarks[0].toString());
```

Comment:
Since the native implementation can return `null` when no results are found I think we should handle this better here. Maybe we can replace this with:

```suggestion
                      var output = 'No results found.';
                      if (placemarks != null && placemarks.isNotEmpty()) {
                        output = placemarks[0].toString();
                      } 
```

We can remove the `print` statement all together.